### ISSUE TITLE
[FR] Scroll timeline when dragging time block to the end of the visible timeline

### ISSUE THREAD

**Issue by FeralFlora** [ivan-lednev/obsidian-day-planner#241]
As is, if I want to reschedule a task, I can only drag it a short distance at a time, because the timeline view does not scroll with my dragging. Therefore, I think it would be very convenient if the timeline view would scroll as you drag a task to the end of the view.

**Comment by ivan-lednev** [comment]
Hello, @FeralFlora, I'll implement this in the future. As a workaround, note that mouse wheel scrolling still works while dragging a task.

**Comment by FeralFlora** [comment]
> As a workaround, note that mouse wheel scrolling still works while dragging a task.

Hello! I see, thanks for the workaround! Btw, I think the autoscroll feature is broken again. Not sure if it should make an issue about that.

**Comment by lukemt** [comment]
Hi!

To be honest this works better for me than to usual drag-scroll-madness where it starts scrolling like crazy just because you move one item a little 😉

**Comment by lukemt** [comment]
btw, the new version is dope! I now started using it again

**Comment by ivan-lednev** [comment]
Hi, @FeralFlora, what do you mean by 'broken'?

**Comment by FeralFlora** [comment]
> Hi, @FeralFlora, what do you mean by 'broken'?

I replied in #243.

**Comment by FeralFlora** [comment]
> Hi, @FeralFlora, what do you mean by 'broken'?

Sorry, I misunderstood you. I see now that you were probably referring to my comment about autoscrolling. Well, it just doesn't do it. I have the autoscroll toggle on, but the scroll doesn't follow the current time.

**Comment by lukemt** [comment]
Hm, interestingly it does work on my installation. I do have the latest version too

**Comment by FeralFlora** [comment]
@lukemt Yeah, that's strange. Here's a demo of it not working for me on the latest version (0.9.0):

![Obsidian_xdqCTii67V](https://github.com/ivan-lednev/obsidian-day-planner/assets/33608083/f300dae7-2671-4f0a-8207-2a224e79514c)

**Comment by FeralFlora** [comment]
Actually, I can't reproduce the autoscroll issue in a test vault, so it's likely some conflict with another plugin or snippet.

**Comment by lukemt** [comment]
No idea... can you try and break it in the Sandbox Vault?

I also use 0.9.0

I'm trying hard to break it. Do you have any tips for me? 

Does restarting obsidian help?

**Comment by FeralFlora** [comment]
> Does restarting obsidian help?

No. I will do some tests with Divide & Conquer later.

**Comment by lukemt** [comment]
Oh, sorry I didn't see your latest message. Ok, great, at least it's reproducible!

**Comment by FeralFlora** [comment]
I think this can be closed, because I can't reproduce the autoscroll issue anymore. Not sure when or why that changed.

**Comment by FeralFlora** [comment]
Actually, this issue wasn't about autoscrolling not working, it was about scrolling the timeline to accomodate dragging time blocks in the timeline, which is still not implemented, so I'm reopening the issue.

**Comment by ivan-lednev** [comment]
Please update and re-check.
